Tests/js/views/ez-searchview.html

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html>
+<head>
+    <title>eZ Search view tests</title>
+</head>
+<body>
+
+<div class="container"></div>
+
+<script type="text/x-handlebars-template" id="searchview-ez-template">
+    <div class="ez-searchview-content">
+        <div class="ez-searchview-search-form">
+            <form class="ez-search-form pure-form">
+                <input class="ez-search-form-input" required type="text" value="">
+                <button type="submit" class="pure-button ez-button">Search</button>
+            </form>
+            <div class="ez-searchlist-content"></div>
+        </div>
+    </div>
+</script>
+
+<script type="text/javascript" src="../../../Resources/public/vendors/yui3/build/yui/yui.js"></script>
+<script type="text/javascript" src="./assets/ez-searchview-tests.js"></script>
+<script>
+    var filter = (window.location.search.match(/[?&]filter=([^&]+)/) || [])[1] || 'raw',
+            loaderFilter;
+    if (filter == 'coverage'){
+        loaderFilter = {
+            searchExp : "/Resources/public/js/",
+            replaceStr: "/Tests/instrument/Resources/public/js/"
+        };
+    } else {
+        loaderFilter = filter;
+    }
+
+    YUI({
+        coverage: ['ez-searchview'],
+        filter: loaderFilter,
+        modules: {
+            "ez-searchview": {
+                requires: [
+                    'ez-templatebasedview',
+                    'array-extras',
+                    'ez-searchlistview',
+                    'node-style',
```

Comment:
Also needed to make the tests pass through setStyle()  ( in _uiSetMinHeight method )